cpp/src/arrow/util/rle_encoding.h

```diff
@@ -209,8 +209,7 @@ class RleEncoder {
     // bit-packed repeated value
     int min_repeated_run_size =
         1 + static_cast<int>(::arrow::bit_util::BytesForBits(bit_width));
-    int repeated_max_size = static_cast<int>(::arrow::bit_util::CeilDiv(num_values, 8)) *
-                            min_repeated_run_size;
```

Comment:
Removed code does seem wrong. Has this caused bugs?

Reply:
Nope, in fact, `static_cast<int>(::arrow::bit_util::CeilDiv(num_values, 8))` has occured twice, so for simplify, I rename it

Reply:
Ahh. This change preserves equivalence.